model/registry.go

```diff
@@ -14,110 +14,46 @@
 
 package model
 
-import (
-	"fmt"
-	"regexp"
-
-	"github.com/golang/protobuf/proto"
-)
+import "github.com/golang/protobuf/proto"
 
 // ConfigKey is the identity of the configuration object
 type ConfigKey struct {
 	// Config object kind, e.g.  "MyKind"
 	Kind string
 	// Config object name, e.g. "my-name"
 	Name string
-	// Config version, e.g. "prod", "canary", "v1".
-	// "default" version is reserved, use ""
-	Version string
+	// Config object namespace, e.g. "default"
```

Comment:
I might stick with convention here (start with export in comment).

Maybe something like: `// Namespace encodes the k8s-specific namespace of the ConfigKey, e.g. "default"`